src/main/java/com/sforce/cd/apexUnit/client/utils/ApexClassFetcherUtils.java

```diff
@@ -432,5 +433,24 @@ public static void logTheFetchedApexClasses(String[] classIds) {
 			}
 		}
 	}
+	
+	/*
+	 * Returns a string array of namespace and class/trigger name
+	 * 
+	 * @param strLine - String value of user provided <NAMESPACE>.<CLASS/TRIGGER>
+	 */
+	public static Map<String, String> parseNamespaceAndName(String strLine){
+		Map<String, String> namespaceAndName = new HashMap<String, String>();
+		String [] parsedName = strLine.split("\\.");
+		if(parsedName.length==1){
+			namespaceAndName.put("name", strLine);
+		}else{
+			namespaceAndName.put("namespace", parsedName[0]);
+			namespaceAndName.put("name", parsedName[1]);
+		}
```

Comment:
Thanks @pmedapuram 
My understanding is we support the formats: Classname or NamespacePrefix.Classname for Manifest file and regex entries. Since NamespacePrefix and Classname cannot contain dot and to prevent users from using multiple dots, I believe we need to validate against using multiple dots and let user know what should be the format to provide.
Examples:
1.
User input: namespace1.class1
Expected output: Consider the namespace and query accordingly to ffilter the classes
2.
User input: namespace1.class1.something.anything
Expected output: Error out saying the format entered is invalid. Acceptable formats are : Classname or NamespacePrefix.Classname

Let me know if you have any questions